src/main/java/com/synopsys/integration/jenkins/scan/global/ApplicationConstants.java

```diff
@@ -42,6 +42,7 @@ public class ApplicationConstants {
     public static final String BRIDGE_BLACKDUCK_SCAN_FAILURE_SEVERITIES_KEY = "bridge_blackduck_scan_failure_severities";
     public static final String BRIDGE_BLACKDUCK_AUTOMATION_FIXPR_KEY = "bridge_blackduck_automation_fixpr";
     public static final String BRIDGE_BLACKDUCK_AUTOMATION_PRCOMMENT_KEY = "bridge_blackduck_automation_prcomment";
+    public static final String BRIDGE_BLACKDUCK_DOWNLOAD_URL_KEY = "BRIDGE_BLACKDUCK_DOWNLOAD_URL";
```

Comment:
So need to check whether we should make this lowercase like other parameters.

Reply:
Done